modules/web-console/e2e/testcafe/environment/envtools.js

```diff
@@ -39,15 +39,14 @@ const insertTestUser = (user = testUser) => {
         .catch((err) => {throw err.message;});
 };
 
-const dropTestDB = (user) => {
+const dropTestDB = (user = testUser) => {
```

Comment:
Please use same signature as in https://github.com/gridgain/gridgain/pull/727/files#r351979491

Reply:
Changed argument to email mask.